tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_container_validation.py

```diff
@@ -355,6 +355,54 @@ def test_valid_containers(
             raw_bytes="ef00 01 01 0004 02 0001 0001 03 0001 0001 04 00",
             validity_error=EOFException.INCOMPLETE_SECTION_SIZE,
         ),
+        Container(
+            # EOF code missing mandatory type section
+            name="EOF1I4750_0001",
+            raw_bytes="ef000102000100010400000000800000fe",
+            validity_error=EOFException.MISSING_TYPE_HEADER,
+        ),
+        Container(
+            # EOF code containing multiple type headers
+            name="EOF1I4750_0002",
+            raw_bytes="ef00010100040100040400000000800000fe",
+            validity_error=EOFException.MISSING_CODE_HEADER,
+        ),
```

Comment:
```suggestion
        Container(
            # EOF code containing multiple type headers
            name="EOF1I4750_0002",
            raw_bytes="ef00010100040100040400000000800000fe",
            validity_error=EOFException.MISSING_CODE_HEADER,
        ),
        Container(
            # EOF code containing multiple type headers, second one matches code length
            name="EOF1I4750_0002_2",
            raw_bytes="ef00010100040100010400000000800000fe",
            validity_error=EOFException.MISSING_CODE_HEADER,
        ),
```
Similar to previous one but second type header's size matches the code section size so the body length matches the header total length.